tasks/copy_to_archive_adapter/bin/run_integration_tests.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+## =============================================================================
+## NAME: run_integration_tests.sh
+##
+##
+## DESCRIPTION
+## -----------------------------------------------------------------------------
+## Runs the integration tests.
+##
+##
+## USAGE
+## -----------------------------------------------------------------------------
+## bin/run_integration_tests.sh
+##
+## This must be called from the (root) directory tasks/copy_to_archive_adapter
+## =============================================================================
+
+## Set this for Debugging only
+#set -ex
+
+## Make sure we are calling the script the correct way.
+BASEDIR=$(dirname $0)
+if [ "$BASEDIR" != "bin" ]; then
+  >&2 echo "ERROR: This script must be called from the root directory of lambda task[bin/run_integration_tests.sh]."
+  exit 1
+fi
+
+
+source ../../bin/common/check_returncode.sh
+source ../../bin/common/venv_management.sh
+
+
+## MAIN
+## -----------------------------------------------------------------------------
+run_and_check_returncode "create_and_activate_venv"
+trap 'deactivate_and_delete_venv' EXIT
+run_and_check_returncode "pip install -q --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org"
+
+## Install the requirements
+pip install -q -r requirements.txt --trusted-host pypi.org --trusted-host files.pythonhosted.org
+check_returncode $? "ERROR: pip install encountered an error."
+
+
+## Check code formatting and styling
```

Comment:
If you want to make it simpler, you can remove the checks that also exist in run_tests.sh.
I am curious how the validation libraries are being installed if they aren't referenced in requirements.txt

Reply:
maybe because the bin/run_integration_tests.sh runs this 
`pip install -q -r requirements.txt --trusted-host pypi.org --trusted-host files.pythonhosted.org`


yes I should remove the checks since bin/run_tests.sh already does that.

Reply:
looks like it only installs the requirements.txt and not the dev one. Interesting!!